core/statistics.c

```diff
@@ -15,6 +15,8 @@
 #include "divelist.h"
 #include "statistics.h"
 
+#include <stdio.h>
```

Comment:
I believe this is only needed for `asprintf()`

Reply:
Agreed. This was an artifact to when I was using asprintf. I will remove it.